lib/OpenQA/Utils.pm

```diff
@@ -558,5 +560,23 @@ sub job_groups_and_parents {
     return \@res;
 }
 
+sub wait_with_progress {
+
+    my $interval = ceil(shift);
+    my $tics;
+    $| = 1;
+
+    do {
+        $tics++;
+        sleep(1);
+        print ".";
+
+    } while ($interval > $tics);
+
+    $| = 0;
+
+    print "\n";
+}
```

Comment:
What are all these blank lines for? Did you change your screen to portrait mode? ;-) Honestly, IMHO you should delete *all* blank lines